.gitignore

```diff
@@ -1,12 +1,4 @@
-*.pyc
-*.swo
-*.swp
-.python-version
+*.**
```

Comment:
That's probably not a good idea. Why did you remove the ignore entries?

Reply:
oh sorry. I was just testing. Didn't mean to commit it.